lm_eval/base.py

```diff
@@ -342,29 +342,55 @@ def _collate(x):
 
         re_ord = utils.Reorderer(requests, _collate)
 
-        for context, until in tqdm(re_ord.get_reordered()):
-            if isinstance(until, str):
-                until = [until]
+        print(re_ord)
 
-            (primary_until,) = self.tok_encode(until[0])
+        if len(re_ord.get_reordered()[0]) == 5:
```

Comment:
This is the MLSUM specific part, right? What does it do?

Reply:
yes. To call model_generate for mlsum I set the values for those 2 extra default arguments, so there are 5 items and for other tasks we don't set those arguments, so it goes to else.